scripts/grt/export.py

```diff
@@ -277,6 +278,47 @@ def annotate(label, human_label=None):
     handle_job.close()
     annotate('export_jobs_end')
 
+    annotate('export_datasets_start', 'Exporting Datasets')
+    handle_datasets = open(REPORT_BASE + '.datasets.tsv', 'w')
+    handle_datasets.write('\t'.join(('job_id', 'dataset_id', 'extension', 'file_size', 'name', 'param_name')) + '\n')
+    for offset_start in range(last_job_sent, end_job_id, args.batch_size):
+        logging.debug("Processing %s:%s", offset_start, min(end_job_id, offset_start + args.batch_size))
+        for job in sa_session.query(model.Job) \
+                .filter(model.Job.id > offset_start) \
+                .filter(model.Job.id <= min(end_job_id, offset_start + args.batch_size)) \
+                .all():
+
+            # If the tool is blacklisted, exclude everywhere
+            if job.tool_id in blacklisted_tools:
+                continue
+
+            input_datasets=(job.input_datasets)
+            for input_dataset in input_datasets:
```

Comment:
we should log output datasets as well, analogous to to the comment below, we have `job_to_output_dataset` table with all the data